p2panda-net/src/engine/mod.rs

```diff
@@ -14,7 +14,7 @@ use futures_util::future::{MapErr, Shared};
 use futures_util::{FutureExt, TryFutureExt};
 use iroh_gossip::net::Gossip;
 use iroh_net::{Endpoint, NodeAddr};
-use p2panda_sync::traits::SyncProtocol;
+use p2panda_sync::SyncProtocol;
```

Comment:
Small thing: How about naming this to `Sync`? Then it's similar to the `Discovery` trait (which is not `DiscoveryProtocol`)

Reply:
The only thing about that is that it conflicts with `Sync` from rust `std`, not sure if that's a problem though... What do you think?